src/index.ts

```diff
@@ -155,6 +172,14 @@ export function withSpid(
       );
     })
     .map(spidStrategy => {
+      if (appConfig.startupIdpsMetadata) {
```

Comment:
maybe you want to store the value of `fromNullable(appConfig.startupIdpsMetadata)` so you can map it here

Reply:
Fixed in 031a6f1